client/my-sites/plans-features-main/index.tsx

```diff
@@ -588,6 +590,7 @@ const PlansFeaturesMain = ( {
 				loggedInFreePlan: {
 					status: 'enabled',
 				},
+				isEligibleForTrial,
```

Comment:
This is not an "action override". Please see the respective type `PlanActionOverrides` and follow what @jdc91 mentioned in the other comment. We currently pass the "callback" and "text" if those are being overwritten, and for the additional text (below the button), you can update `PlanActionOverrides` type to include `footnote` or `postButtonText` property.